registry-scanner/README.md

```diff
@@ -0,0 +1,46 @@
+# Registry Scanner
+
+## Introduction
+
+ArgoCD Image Updater provides functionalities that can be reused by other projects, most notably the feature to inspect OCI and Docker registries' contents and pick an image based on some constraints.
+
+The registry-scanner is a reusable library for registry inspections and fetching images based on the configured strategy. Having registry-scanner as a separate library allows users to integrate just the registry-scanner.
+
+
+## Current status
+
+Registry Scanner is under active development. We would not recommend it
+yet for *critical* production workloads, but feel free to give it a spin.
+
+We're very interested in feedback on usability and the user experience as well
+as in bug discoveries and enhancement requests.
+
+**Important note:** Until the first stable version (i.e. `v1.0`) is released,
+breaking changes between the releases must be expected. We will do our best
+to indicate all breaking changes (and how to un-break them) in the
+[Changelog](CHANGELOG.md)
```

Comment:
I'd replace this by a statement that this component is not yet ready for public consumption, and that we're currently working on a first release.